src/glb-director/glb_fwd_config.c

```diff
@@ -79,7 +86,11 @@ struct glb_fwd_config_ctx *create_glb_fwd_config(const char *config_file)
 		goto fail;
 	}
 
+#ifndef NO_DPDK
```

Comment:
These could be macros defined as appropriate (using #ifdef in the macro definition for example) so that the code need not be peppered with the #ifdef etc. Even declarations further down could be absorbed into similar macros.